### ISSUE TITLE
Markdown: Headline syntax ignored

### ISSUE THREAD

**Issue by jefflembeck** [npm/marky-markdown#324]
Original issue by @mk-pmb

https://github.com/npm/www/issues/103

> In my README.md, three number signs (###) used to start a headline, and it still works on Github, but today it broke on npmjs.com. There it shows as just a regular paragraph that has number signs as its first word.

**Comment by revin** [comment]
Thanks @jefflembeck. This is basically the same issue as #317:

```markdown
<a class="readme-ssi-toc-target" id="toc-rxu-rgx-text" name="toc-rxu-rgx-text"></a>
### rxu(rgx, text) &nbsp; &nbsp; [☂](test/match.js)
Try to match RegExp `rgx` on string `text`, with fancy extras.
Returns the match if there was one, or `false` otherwise.
The latter is chosen so you can safely access number properties
of `rxu()`'s result.
```

Notice the lack of blank line between the `html_block` and what we want to be interpreted as a `heading_open`.

**Comment by revin** [comment]
So I have to retract my previous statement about this being the same as that other issue. It's technically different. The other one, we had an `html_block` that was being greedier than github's parsing; in this one it's a paragraph with some `html_inline` followed by some text that gets parsed as being part of the paragraph because there's no blank line.

The fix will be similar in spirit to #325.

**Comment by revin** [comment]
Hah, this is actually a bug in the ["no indented headings" plugin](https://github.com/npm/marky-markdown/blob/d9a97ee62be5985678f8f1d90c1db1ca99ea06a9/lib/gfm/indented-headings.js#L23); we were accidentally dropping the `alt` property from the parser rule, but that's the thing that tells markdown-it which rules can interrupt which other rules. So headings weren't being allowed to interrupt paragraphs anymore, and the HTML was just a red herring.
